src/StoreApi/Utilities/RateLimits.php

```diff
@@ -0,0 +1,141 @@
+<?php
+namespace Automattic\WooCommerce\StoreApi\Utilities;
+
+use WC_Rate_Limiter;
+use WC_Cache_Helper;
+
+/**
+ * RateLimits class.
+ */
+class RateLimits extends WC_Rate_Limiter {
+
+
+	/**
+	 * Amount of max requests allowed for the defined timeframe.
+	 *
+	 * @var int
+	 */
+	const LIMIT = 25;
+
+	/**
+	 * Time in seconds before rate limits are reset.
+	 *
+	 * @var int
+	 */
+	const SECONDS = 10;
```

Comment:
Should these be filtered so the merchant can modify them at will?

Reply:
Might be interesting, since this is not a feature for the average merchant, that might want more control over it.
@senadir what do you think about @opr 's suggestion?

Reply:
Ok, so these will be filtered and condensed into having only one filter for options (proxy support, limits..) 😎

Reply:
I don't understand, sorry! Are you suggesting we'd have only one filter to control three different variables? 🤔

Reply:
Yes, instead of individual filter for each value, a single filter that for all of them should work.